scripts/trufflehog-check.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -e
+
+if ! [ -x "$(command -v truffleHog)" ]; then
+    echo The command truffleHog is not available, installing...
+
+    # Install truffleHog in a venv
+    python -m venv trufflehog-check
+    source trufflehog-check/bin/activate
+    python -m pip install --upgrade pip
+    pip install truffleHog
+else
+    echo The command truffleHog is available
+fi
+
+echo "trufflehog-check.sh: Scanning repo ..."
+
+last_merge=$(git log --format=format:"%H" -n 1 raft-tdp-main)
+
+# $1 - The first argument to this script, the current git branch name
+# --since_commit - Look at all commits since the last merge into raft-tdp-main
+# --entropy=True - Entropy checks on large git diffs
+python ./trufflehog-check/lib/python3.8/site-packages/truffleHog/truffleHog.py \
+  --regex \
+  --entropy=True \
```

Comment:
Does the since_commit limiting significantly lower the output while entropy is True? I remember this being a problem before, so if so glad to see that we could still use this.

Reply:
Correct, with the much smaller number of commits being scanned the entropy check is no longer a problem and has a usable volume of output.